### PR TITLE
Add Redis migration example

### PR DESCRIPTION
**Pull request by stano45**
Added an example to demonstrate the migration of a Redis container with 2 active connections.

- Added an HTTP server (app) to fetch data from Redis via the load balancer
- Added a producer that increments a counter in Redis via the load balancer
- Added a client that periodically fetches and prints data from the HTTP server (this includes uptime of the Redis instance and # of active connections)

https://github.com/user-attachments/assets/eeb1dd35-645f-4fb4-948c-e69ac4df6b38

@rst0git PTAL

### CODE REVIEW

File: examples/redis/build.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+printf "\n-----Creating switch-----\n"
+sudo podman run -d \
+    --name s1 \
+    --privileged \
+    --replace \
+    --network host \
+    -v ../../examples/switch_container/s1.sh:/s1.sh \
+    -v ../../load_balancer/build/load_balance.json:/load_balance.json \
+    --entrypoint /s1.sh \
+    p4c
```
Comment (rst0git) [inline examples/redis/build.sh:57]: How do you create the `p4c` container image? Or is this [`p4lang/p4c`](https://hub.docker.com/r/p4lang/p4c) ?

Comment (stano45) [reply examples/redis/build.sh]: Yes, this is `p4lang/p4c`. I updated the scripts with the proper image name.

**Review by rst0git — COMMENTED** [review]
It might be useful to also add a README file for this example (similar to the other examples).

**Comment by rst0git** [comment]
LGTM

**Pull request merged**